shared/styles/colors.js

```diff
@@ -17,34 +17,29 @@ const colors = {
   black_60_on_white: 'rgb(102,102,102)',
   black_75: 'rgba(0, 0, 0, 0.75)',
   black_75_on_white: 'rgb(64,64,64)',
-  blue: '#33a0ff',
-  blue2: '#66b8ff',
-  blue3: '#a8d7ff',
-  blue3_20: 'rgba(168, 215, 255, 0.2)',
-  blue3_20_on_white: 'rgb(238, 247, 255)',
-  blue3_40: 'rgba(168, 215, 255, 0.4)',
-  blue3_40_on_white: 'rgb(220, 239, 255)',
-  blue3_60: 'rgba(168, 215, 255, 0.6)',
-  blue3_60_on_white: 'rgb(203, 231, 255)',
-  blue4: '#ebf5fc',
-  blue5: '#F8FAFD',
+  blue: '#4C8EFF',
```

Comment:
where did my on_white versions go, we need this for mobile perf

Reply:
Ah - thought it was outdated - putting them back